source/blog/2018-01-16-ovirt-report-using-grafana.html.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Build oVirt Report Using Grafana
+author: sradco,
+tags: oVirt, oVirt 4.2, open source virtualization, report, dwh
+date: 2018-01-16 09:00:00 CET
+---
+
+[Grafana](https://grafana.com/), The open platform for beautiful analytics and monitoring,
+adds support for [PostgreSQL](http://docs.grafana.org/features/datasources/postgres/).
+
+It in now possible to connect Grafana to [oVirt DWH](https://www.ovirt.org/documentation/how-to/reports/dwh/),
+in order to visualize and monitor the oVirt environment.
+
+**Grafana dashboard example**
+![](/images/grafana_dashboard_example.png)
+
+READMORE
+
+If you wish to create dashboards to monitor Ovirt environment, you will need to [install Grafana](http://docs.grafana.org/installation/rpm/).
+
+Grafana automatically create and admin [user](http://docs.grafana.org/installation/configuration/#admin-user) and [password](http://docs.grafana.org/installation/configuration/#admin-password).
+
+You will need to add a [PostgreSQL data source](http://docs.grafana.org/features/datasources/graphite/#adding-the-data-source) that connects to the DWH database.
+
+For example:
+![](/images/grafana_data_source_example.png)
+
+Now you can start creating your dashboard widgets.
+
+Go to `Dashboards` -> `+ New`.
+
+
+
+**Graph panel example:**
+
+To add a `Graph` type panel, on the left side you have the [Row controls menu](http://docs.grafana.org/guides/getting_started/#dashboards-panels-rows-the-building-blocks-of-grafana).
+Go to the `+ Add Panel`, and pick `Graph`.
+
+Query example for the - Five Most Utilized Hosts by Memory / CPU:
+
```

Comment:
There's a markdown for raw text?

Reply:
What do you mean?

Reply:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code or https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#blockquotes